app/apps/cases/models.py

```diff
@@ -80,8 +80,10 @@ class Meta:
         to=Address, null=True, on_delete=models.CASCADE, related_name="cases"
     )
 
-    def get(identification):
-        return Case.objects.get_or_create(identification=identification)[0]
+    def get_current_state(self):
+        if self.casestate_set.count() > 0:
```

Comment:
Waar kom de casestate_set vandaan? Ik zie het niet als veld in het model of als property. (Ik zie het ook in de CaseSerializer terugkomen)

Reply:
was django standaard nu veranderd naar case_states